src/Microsoft.ML.PCA/PcaTransformer.cs

```diff
@@ -36,7 +36,7 @@ namespace Microsoft.ML.Transforms.Projections
     /// <include file='doc.xml' path='doc/members/member[@name="PCA"]/*' />
     public sealed class PcaTransformer : OneToOneTransformerBase
```

Comment:
>PcaTransformer [](start = 24, length = 14)

Can we rename it PrincipalComponentAnalysisTransformer?

Reply:
Yes I agree!

---
In reply to: [254044312](https://github.com/dotnet/machinelearning/pull/2366#discussion_r254044312) [](ancestors = 254044312)